cookbooks/continuous_delivery.rst

```diff
@@ -0,0 +1,187 @@
+Continuous Delivery
+===================
+
+Continuous Delivery is an approach that aims at releasing softwares faster and
```

Comment:
`softwares` should be `software` according to https://en.wiktionary.org/wiki/softwares